Minio/ApiEndpoints/BucketOperationsExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reactive.Linq;
+using Minio.DataModel;
+using Minio.DataModel.Args;
+
+namespace Minio.ApiEndpoints;
+
+public static class BucketOperationsExtensions
+{
+    // Used for backwards compatibility
+    [Obsolete("Use the IAsyncEnumerable version instead")]
+    public static IObservable<Item> ListObjectsAsync(this IBucketOperations self, ListObjectsArgs args)
```

Comment:
This method was added to provide compatibility with clients that rely on `IObservable`.